docs/tutorials/getting-started.md

```
@@ -33,25 +33,35 @@ The `workspace/` directory is the runtime configuration volume. Deployment-ident
+
+# External A2A registry (for remote agents)
 cp workspace/agents.yaml.example   workspace/agents.yaml
 cp workspace/projects.yaml.example workspace/projects.yaml
 cp workspace/discord.yaml.example  workspace/discord.yaml
 cp workspace/google.yaml.example   workspace/google.yaml
 ```
 
-Then fill in your URLs, channel IDs, and env var names. The directory layout:
+Then fill in your model aliases, system prompts, URLs, and channel IDs. The directory layout:
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)